Docs/Email.README.md

```diff
@@ -0,0 +1,26 @@
+# AWS Simple Email Service (SES) - Email Sending Service
+
+The SES is deployed and managed in the infrastructure project so all Virtual Finland projects can easily send emails from organization domain using SES client.
+
+## Deployment dependencies
+
+The SES depends on a domain zone being registered in AWS Route53. Currently\* the domain zone is managed by the [Access Finland MVP](https://github.com/Virtual-Finland-Development/access-finland) project and thus there's a circular dependecy between the infrastructure and access-finland projects: the access-finland requires infrastructure to be deployed first and the infrastructure requires access-finland to be deployed before the SES can be deployed.
+
+There's also a dependency between the different deployment stacks in the project. For example when the domain zone is created in the `mvp-staging` stack, but the `dev` stack also needs to send emails, then the `dev` infrastructure-stack needs to be deployed after the `mvp-staging` stack. 
```

Comment:
Again, sama kun ylhäällä! Mutta siis ymmärsinkö oikein, että mvp-staging "owns" dev stackin mitä domain konffaukseen tulee, niin siksi tämä järjestyst. Eli `ses:this-stack-owns-other-domains`

Reply:
Uusin silmin (ei konkreettisesti) luin noita muuttujanimiä ja menin itsekin sekaisin. Aikaisemmassa toteutusajatuskompleksiossa oli tarve että olisi kaksi suuntainen dependecy-vaade että esim. deviin deplauksen tarvitsisi tietää missä domain oikeasti sijaitsee, ja näin ei siis olekaan. 

Selkeytyin tuota että nyt on uudet konffit `ses:create-domain-records: <boolean>` ja `ses:also-create-domain-records-for: <string[]>`.

Reply:
> Oiskohan tässä (tai jossain muualla) hyvä listata nämä riippuvuudet, jotta ois helpompaa ja nopeampaa löytää vs. käy pulumin konffeista vakoilemassa. Tietysti tavallaan tää on vaan meille relevanttia tietoa, kun on monia eri ympäristöjä tässä kohtaa.
>
>Ja jos tuota fresh tuotantodeplausta miettii, niin meneekö vielä yksiin nykyisen ohjeen mukaan deplaukset?

Joo mietin että tuonne prodects-repositoriin viimeistään pitäisi päivitellä, mutta siihen dokkariin ja muutenkin fresh-deplaukseen vaikuttaa toi että josko siirtäisi vielä tuon domainin luonnin kanssa tähän infra-projektiin, ja sitä vielä vähän voi puntaroida että oliskohan intoa, niin vielä siltä osin dokumentointi jääköön keskeneräiseksi!

Reply:
Tuo uudellen nimetty on paljon selkeempi!